proxy/stats_get_prediction.go

```diff
@@ -15,9 +15,15 @@ import (
 
 const StatsGetWorkers = 5
 
+type StatsGetDef struct {
```

Comment:
I feel like it's better to just standardize on generating and passing around `StatsGetTask` instead of duplicating half the data in a separate struct.

Reply:
Made a new commit that only uses `StatsGetTask`.